src/ucs/type/spinlock.h

```diff
@@ -18,16 +19,26 @@ BEGIN_C_DECLS
 /** @file spinlock.h */
 
 
+/* Spinlock static initializer */
+#define UCS_SPINLOCK_INITIALIZER {UCS_SPINLOCK_FREE}
+
+
 /* Spinlock creation modifiers */
 enum {
     UCS_SPINLOCK_FLAG_SHARED = UCS_BIT(0) /**< Make spinlock sharable in memory */
 };
```

Comment:
minor: can be deleted

Reply:
i think we can't, because it is UCS installable header

Reply:
we can't delete it?

Reply:
would not it break compatibility then?

Reply:
UCS is not compatible currently

Reply:
removed flags (constant and parameter)